### ISSUE TITLE
Working together and benefit from each other work

### ISSUE THREAD

**Issue by illwieckz** [cagelight/dogiradiant#1]
Hi, I just discovered dogiradiant (and its fork upgRadiant). It looks like I'll have to add two mores to my [know list of radiants](https://dl.illwieckz.net/b/radiant/known-radiant-list.md):

- AARadiant (from Alien Arena project, NetRadiant fork)
- CoDRadiant (from Call of Duty project)
- DarkRadiant (from The Dark Mod project, for Doom 3 engine based games, GtkRadiant 1.5.0 fork)
- Embrace (from Vampire: The Masquerade – Redemption project, part of Nod SDK, QERadiant fork)
- ETXRadiant (from ET:XreaL project, DarkRadiant fork)
- GtkRadiant (the 1.4/1.6 branch, multiplatform port of QERadiant using GTK gui toolkit instead of MFC one)
- GtkRadiant (the 1.5 branch)
- HMF2Radiant (from Heavy Metal: FAKK2 project, part of FAKKTools, the binary is named Q3Radiant, and advertises itself as QERadiant in some places)
- KpRadiant (from Kingpin: Life of Crime project, remaining web pages makes strong references to QERadiant)
- MacRadiant (GTKRadiant port to Mac OS X)
- MoHRadiant (MOHAATools, from Medal of Honnor: Allied Assault project, QERadiant fork)
- NetRadiant (community fork of GTKRadiant 1.5, advertised as “A Stabilized Q3 Map Editor” in former icculus' repository with a former website hosted by Alien Trap, now hosted and somewhat maintained by the Xonotic team)
- NetRadiant Custom (fork by Garux, hard to merge)
- ODRadiant (from OverDose project, there is some GTKRadiant traces in it)
- OWRadiant (from OWEngine project, a renamed Qio engine reusing OpenWolf name)
- QERadiant (original public radiant based on QE4, QE stands for Quake Ed the in-house editor for Quake¹, QE4 was the in-house Quake 2 level editor)
- Q3Radiant (former name for QERadiant, in-house Quake 3 level editor)
- QioRadiant (from Qio Engine project)
- QtRadiant (the one by Ensiform)
- QtRadiant (the one by Radegast)
- Radiant (from Doom 3 source tree)
- SDRadiant (from Wolfenstein: Enemy Territory project, SD stands for Splash Dammage)
- SoFRadiant (from Soldier of Fortune project)
- ÜberRadiant (from Star Trek: Elite Force 2 project, Über stands for ÜberGames)
- UFORadiant (from UFO:AI project, based on an old DarkRadiant fork)
- VolatileRadiant (from Volatile3D II project, XaeroX said it is based on Q3Radiant, also named VolatileLevelDesignSDK)
- VRadiant (from Vecxis project, NetRadiant fork)
- WolfRadiant (from ET:Legacy project)
- XreaLRadiant (from XreaL project)
- ZeroRadiant (code name for GTKRadiant 1.6 which is a successor to GTKRadiant 1.4)

:grin: :grimacing: :sweat_smile: :upside_down_face: :smile: 

So, anyway, maybe you don't know there is a maintained radiant there:

- https://netradiant.gitlab.io

NetRadiant is the continuation of GtkRadiant 1.5 (while GtkRadiant 1.6 is in fact based on 1.4 for some unknown reasons) and is the editor of choice for idtech based community games.

I've seen you just switched GtkRadiant from Scons to CMake, NetRadiant use CMake for years now! NetRadiant supports many games and I would be very happy to help us to include support for your project. The forge is there: https://gitlab.com/xonotic/netradiant

NetRadiant is currently maintained by [Xonotic](https://xonotic.org/) and [Unvanquished](https://unvanquished.net/) guys (I'm myself coming from Unvanquished) and we review and merge contributions from other projects.

Also the q3map2 shipped with netradiant is much more advanced than the gtkradiant one (which is missing many years of developments). I hope to see you soon! :slightly_smiling_face:

**Comment by cagelight** [comment]
Appreciate the concern, but this project (dogiradiant) and its derivative (upgRadiant) are just two friends messing around and learning with what we're already familiar with. These are NOT real projects and nobody is ever expected to use them except us.

**Comment by illwieckz** [comment]
I've read that DogiJK is looking at implementing navigation meshes using recastnavigation.

You may be interested in the fact there is already a q3map2 navmesh generation code based on recastnavigation (for idtech3-based engine using recastnavigation like [Daemon](https://github.com/DaemonEngine/Daemon)), the code is there and is written in a way it its technically possible to merge it in upstream q3map2: https://github.com/DaemonEngine/daemonmap

Even with two friends you better not underestimate your project. And you'll go farther if you take a train at first. :slightly_smiling_face:

**Comment by cagelight** [comment]
I'm curious about this "NetRadiant" project, but it seems to currently have a fatal issue when I try to load the JA gamepack:
![Screen Capture_netradiant_20200919034809](https://user-images.githubusercontent.com/3761279/93662089-36e90500-fa2b-11ea-83d1-200451d3a44f.png)

Sorry to say, but stuff like this is why I generally like to stick with what I know, even if it's not the most "current" iteration.

**Comment by illwieckz** [comment]
Given the picture the issue is in a file that is part of the gamepack, a collection of configuration files usually provided by the game.

I would like to see NetRadiant working with that OpenJK gamepack, either fixing NetRadiant, either fixing that gamepack. If it's not for you it would be for the others. :slightly_smiling_face:

I'll look at it one day, maybe I can ask SomaZ about some required OpenJK information. Maybe the build tool just not downloads the last version of the gamepack…

Basically some gamepacks may require some fixes because of all those forks reimplement the same things every time because almost every one underestimates the preciousness of their work. :smile: 

So I'll look at it, thanks for the report!
